scripts/test_parser.py

```diff
@@ -0,0 +1,102 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+import unittest
```

Comment:
Much needed! Thanks for adding this unit test module!

Nit: should we renamed it to `test_cbmc_parser.py` to make it more clear?

Reply:
Sure